rubix-bookkeeper/pom.xml

```diff
@@ -128,6 +134,7 @@
                                     <include>com.google.guava:guava</include>
                                     <include>com.readytalk:metrics-statsd-common</include>
                                     <include>com.readytalk:metrics3-statsd</include>
+                                    <include>io.dropwizard.metrics:metrics-jvm</include>
```

Comment:
Are we shading the jar here ? If yes, can we put a relocation for this so that we can differentiate this dependency from other dropwizard jar

Reply:
Added relocation